test-runner/src/index.ts

```diff
@@ -81,7 +81,7 @@ export async function runTests(config: RunnerConfig, suite: Suite, reporter: Rep
   // Trial run does not need many workers, use one.
   const jobs = (config.trialRun || config.debug) ? 1 : config.jobs;
   const runner = new Runner(suite, { ...config, jobs }, reporter);
-
+  fs.mkdirSync(config.outputDir, { recursive: true });
```

Comment:
Let's also clean it before running?

Reply:
It can't because the index file is required by every test worker. This should be in cli right?

Reply:
I decided to clean it on the call site, otherwise can be unexpected...